### ISSUE TITLE
DBIC:DH:DeployMethod::SQL::Translator breaks literal MySQL DDL

### ISSUE THREAD

**Issue by augensalat** [frioux/DBIx-Class-DeploymentHandler#72]
Method `_split_sql_chunk` breaks SQL at what it thinks is a statement delimiter,
and removes what it thinks is a transaction start.

Consider this MySQL code:
```
DELIMITER $$

CREATE PROCEDURE `hello_world`
BEGIN
  SELECT 'Hello World!';
END$$

DELIMITER ;
```
Changing the delimiter is required for this to work but DH has no idea about that.
And `BEGIN` is not necessarily the start of a transaction.

**Comment by augensalat** [comment]
I played around with `SQL::SplitStatement` which seems to do a pretty good job in splitting SQL.
Another solution is not splitting the SQL statements at all for DBD modules that support multiple staments, like `DBD::mysql` does at least.

**Comment by augensalat** [comment]
Oh... `t/10-split-sql-chunk.t` explicitly tests that `DBIx::Class::DeploymentHandler::DeployMethod::SQL::Translator` *really* breaks SQL:
Make sure to remove `BEGIN` from a `FUNCTION` declaration!
Yes, works. Go ahead!

**Comment by augensalat** [comment]
Fixed in v0.002233
